app/services/selection.js

```diff
@@ -200,11 +206,24 @@ export default Service.extend({
       const inSelection = selected.features.find(selectedFeature => selectedFeature.properties.geoid === properties.geoid);
 
       if (inSelection === undefined) {
-        selected.features.push({
-          type,
-          geometry,
-          properties,
-        });
+
+        if (this.get('summaryLevel') === 'boroughs' || this.get('summaryLevel') === 'cdtas') {
+          const currentGeographyTable = SUM_LEVEL_DICT[this.get('summaryLevel')].sql;
+          // Temporary patch: ensure entire geography geojson is used, not just the geometry within clicked tile. 
+          const sqlQuery = `
+            SELECT * FROM (${currentGeographyTable}) a WHERE geoid = '${properties.geoid}'
+          `;
+
+          this.getEntireGeoTask.perform(sqlQuery, (json) => {
+            this.set('current', { ...selected, features: selected.features.concat(json.features)});
```

Comment:
Do we know if there's a risk of this leading to duplicate features in the final result? By duplicate I mean a cut-off feature and a "correct" feature for geos on the border of the tile

Reply:
It shouldn't because of `inSelection` check  above at https://github.com/NYCPlanning/labs-factfinder/pull/913/files#diff-1cd4e650bab88d311ed88a7b3258947d3e0e3f7edd80ddf55603a96602a9d695R206